draft-ietf-scitt-architecture.md

```diff
@@ -417,12 +418,26 @@ Transparency Services MUST support at least one of these methods:
 
 ### Append-only Log
 
-The security properties of the Append-only Log are determined by the choice of the verifiable data structure used to produce Receipts.
+The security properties of the Append-only Log are determined by the choice of the verifiable data structure used by the Transparency Service to produce Receipts. 
+This verifiable data structure MUST support the following security requirements:
 
-In addition to Receipts, some verifiable data structures might support additional proof types, such as proofs of consistency, or proofs of non inclusion.
+Append-Only: 
+
+: once included in the verifiable data structure, a Signed Statement cannot be modified, deleted, or reordered; hence its Receipt provides a universally-verifiable proof of registration. 
```

Comment:
I can imagine readers might get hung up on "universally-verifiable proof of registration".

I wonder if we should define SCITT receipts as "proof of registration" that build on COSE receipts, which are signed inclusion proofs.

Reply:
I like 'offline verifiable' a lot more than 'universally'

Reply:
> I like 'offline verifiable' a lot more than 'universally'

Suggested change added

Reply:
This just isn't true tho is it ? If a service has a take down request or whatever, it is legaly obliged to destroy the statement. This is usually accomplished by deleting the pre-image or "forgeting" some encrypting key

Reply:
I also prefer offline verifiable though

Reply:
Resolving as I've merged in the _offline verifiable_ change. Please re-open if folks don't agree.